docs/index.html

```diff
@@ -145,29 +145,14 @@ <h3>Map Controls</h3>
     <div class="no-break">
         <h3>Browser Support</h3>
 
-        <h4>Desktop</h4>
-
         <ul>
             <li>Chrome</li>
             <li>Firefox</li>
-            <li>Safari 5+</li>
-            <li>Opera 12+</li>
-            <li>IE 9&ndash;11</li>
+            <li>Safari</li>
             <li>Edge</li>
         </ul>
     </div>
 
-    <div class="no-break">
-        <h4>Mobile</h4>
```

Comment:
Removed the sections for mobile and desktop as we want to support the same browsers on both.